atom/browser/api/atom_api_session.cc

```diff
@@ -345,6 +390,32 @@ void OnClearStorageDataDone(const base::Closure& callback) {
 
 void OnClearHistory() {}
 
+void OnGetHistory() {}
+
+void OnGetHistoryB(history::QueryResults* qr) {
+
+  v8::Isolate* isolate = v8::Isolate::GetCurrent();
+
+  if (!isolate) {
+    LOG(ERROR) << "null isolate";
+    return;
+  }
+
+  node::Environment* env = node::Environment::GetCurrent(isolate);
+  if (!env) {
+    LOG(ERROR) << "null env";
+    return;
+  }
+
+  auto event = v8::Local<v8::Object>::Cast(mate::Event::Create(isolate).ToV8());
+
+  mate::EmitEvent(isolate,
```

Comment:
actually do we need this at all? If we care about the results I think we should use the callback

Reply:
maybe make this an emptyfn callback?